charts/ingressmonitorcontroller/README.md

```diff
@@ -11,6 +11,8 @@ Source code can be found [here](https://github.com/stakater/IngressMonitorContro
 To install IMC helm chart run the following
 
 ```sh
+# Install CRDs
+kubectl apply -f https://raw.githubusercontent.com/stakater/IngressMonitorController/master/deploy/crds/endpointmonitor.stakater.com_endpointmonitors_crd.yaml
```

Comment:
CRDs are installed when you install the chart. You don't need to do this step manually